desktop/app/splash.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+// Checks to see if we've launched before (we touch started.txt). If not, write the file and open a browser
+
+const helpURL = 'https://keybase.io/docs/cli_kbstage'
+const filePath = 'started.txt'
```

Comment:
I'm not sure what'll happen on OS X; that sounds plausible.

But on Linux someone might launch it by typing "keybase" from a shell in their home dir instead, and then I think they'd get $HOME/started.txt.  Or, if it uses the app's own directory on Linux, we'd fail to write the file because the package will have been installed as root, and their user won't have permission to write there.